contracts/interfaces/CErc20Interface.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.5.17;
```

Comment:
I think it should be in "contracts/providers/compound/CErc20Interface.sol".

Reply:
I agree - seems Compound-specific

Reply:
The reason I put it in the interfaces folder is because 'PairAggregatorInterface' is in the interfaces folder too (which is chainlink). Happy to move it but was going for consistency